languages/common-lisp/concepts/default-parameters/about.md

```diff
@@ -1 +1,9 @@
-TODO: add information on default-parameters concept
+An optional parameter is designated by the `&optional` lambda list keyword in a lambda list. Optional parameters are not required, can have a default and also can specify a "supplied-p parameter" which will be "true" or "false" depending on whether an argument was provided for the parameter.
+
+Arguments are consumed for optional parameters after any arguments are consumed for required parameters and before any other types.
```

Comment:
This might be a bit clearer with some specifics and rewording, maybe just saying that optional parameters come after required ones, but before keywords and rest parameters (then we could maybe link to those concepts)